onlinejudge/service/anarchygolf.py

```diff
@@ -61,9 +61,9 @@ def _parse_sample_tag(self, tag: bs4.Tag) -> Optional[Tuple[str, str]]:
             while nxt and nxt.string.strip() == '':
                 nxt = nxt.next_sibling
             if nxt.name == 'pre':
-                s = utils.textfile(utils.dos2unix(nxt.string.lstrip()))
+                s = utils.textfile(utils.dos2unix(utils.parse_content(nxt).lstrip()))
             else:
-                s = ''
+                s = utils.textfile('')
```

Comment:
```suggestion
                s = ''  # *NOTHING* means  that the empty string "" is input, not "\n".
```

`*NOTHING*`means the empty string. You can check this by submitting`xxd` command via the `bash` language.
(I should leave this as a comment :bow:)

Reply:
You're right. Thanks!